File: OOP/C sharp/QuanLyTourDuLich/QuanLyTourDuLich/PHANCONG.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QuanLyTourDuLich
{
    public class PHANCONG
    {
        public string MaSoNV { get; set; }
        public string MaSoTour { get; set; }
        public NHANVIEN NhanVien { get; set; }
        public TOUR Tour { get; set; }
        public PHANCONG(string MaSoNV, string MaSoTour, NHANVIEN NhanVien, TOUR Tour)
        {
            this.MaSoNV = MaSoNV;
            this.MaSoTour = MaSoTour;
            this.NhanVien = NhanVien;
            this.Tour = Tour;
        }

        public PHANCONG(PHANCONG phancong)
        {
            MaSoNV = phancong.MaSoNV;
            MaSoTour = phancong.MaSoTour;
            NhanVien = phancong.NhanVien;
            Tour = phancong.Tour;
        }

    }
}

File: OOP/Java/QuanLyTourDuLich/src/mypack/PHANCONG.java
package mypack;

public class PHANCONG
{
    public String MaSoNV ;
    public String MaSoTour ;
    public NHANVIEN NhanVien ;
    public TOUR Tour ;
    public PHANCONG(String MaSoNV, String MaSoTour, NHANVIEN NhanVien, TOUR Tour)
    {
        this.MaSoNV = MaSoNV;
        this.MaSoTour = MaSoTour;
        this.NhanVien = NhanVien;
        this.Tour = Tour;
    }
    public PHANCONG(PHANCONG phancong)
    {
        MaSoNV = phancong.MaSoNV;
        MaSoTour = phancong.MaSoTour;
        NhanVien = phancong.NhanVien;
        Tour = phancong.Tour;
    }
	public String getMaSoNV() {
		return MaSoNV;
	}
	public void setMaSoNV(String maSoNV) {
		MaSoNV = maSoNV;
	}
	public String getMaSoTour() {
		return MaSoTour;
	}
	public void setMaSoTour(String maSoTour) {
		MaSoTour = maSoTour;
	}
	public NHANVIEN getNhanVien() {
		return NhanVien;
	}
	public void setNhanVien(NHANVIEN nhanVien) {
		NhanVien = nhanVien;
	}
	public TOUR getTour() {
		return Tour;
	}
	public void setTour(TOUR tour) {
		Tour = tour;
	}
    
}
